pkg/tsdb/mysql/sqleng/sql_engine.go

```diff
@@ -300,14 +304,14 @@ func (e *DataSourceHandler) executeQuery(query backend.DataQuery, wg *sync.WaitG
 	}
 
 	if err := convertSQLTimeColumnsToEpochMS(frame, qm); err != nil {
-		errAppendDebug("converting time columns failed", err, interpolatedQuery)
+		errAppendDebug("converting time columns failed", err, interpolatedQuery, backend.ErrorSourcePlugin)
 		return
 	}
 
 	if qm.Format == dataQueryFormatSeries {
 		// time series has to have time column
 		if qm.timeIndex == -1 {
-			errAppendDebug("db has no time column", errors.New("no time column found"), interpolatedQuery)
+			errAppendDebug("db has no time column", errors.New("no time column found"), interpolatedQuery, backend.ErrorSourceDownstream)
```

Comment:
@zoltanbedi Yes, this is user error when they select time series format and their query does not include time column.
<img width="1318" alt="image" src="https://github.com/user-attachments/assets/13b1667c-54e7-4896-b5e3-81cfdc5bee23">

@gwdawson could we also improve the error message to make it more actionable and understandable. Something like:

```suggestion
			errAppendDebug("db has no time column", errors.New("time column is missing; make sure your data includes a time column for time series format or switch to a table format that doesn't require it"), interpolatedQuery, backend.ErrorSourceDownstream)
```